README-usage.md

```diff
@@ -28,6 +28,16 @@ encounter while using the cli may guide you through the process.
 ```
 </details>
 
+##### Run Properties
+The `run_properties` contains information that is specific to a particular run of the pipeline.
+Required properties are:
+- `run` uniquely defines the build and contains more info about the run 
+- `parameters` shows the deploy target
+- `versioning` defines an object containing info about the `version`, `branch`, `pr_number` or `tag`. Either `pr_number` or `tag` need to be set
```

Comment:
Mayb it'll e more efficient  to refer to the inline documentation on `mpyl.steps.models.RunProperties` to avoid double bookkeeping.